clippy_lints/src/manual_unwrap_or_default.rs

```diff
@@ -134,7 +134,7 @@ fn handle_match<'tcx>(cx: &LateContext<'tcx>, expr: &'tcx Expr<'tcx>) -> bool {
             "match can be simplified with `.unwrap_or_default()`",
             "replace it with",
             format!("{receiver}.unwrap_or_default()"),
-            Applicability::MachineApplicable,
+            Applicability::MaybeIncorrect,
```

Comment:
Would be a shame to lose `--fix` in cases where it is appropriate, could set the applicability based on https://doc.rust-lang.org/nightly/nightly-rustc/clippy_utils/fn.span_contains_comment.html

Reply:
thanks, forgot that I can do that